src/js/components/AbstractTabsSelection.js

```diff
@@ -0,0 +1,34 @@
+import Control from './Control';
+import Message from '../constant/Message';
+export default class AbstractTabsSelection extends Control {
```

Comment:
Shouldn't define a whole new class AbstractTabsSelection. Only class Tabs use this.